From a0c9e2dddb744ad78bc7f15021d07628d00d8915 Mon Sep 17 00:00:00 2001
From: Pan Ortiz <panjeoc@proton.me>
Date: Tue, 12 Nov 2024 02:53:14 +0000
Subject: [PATCH 45/56] arch: arm64: boot: dts: qcom: sm8150-xiaomi-nabu.dts:
 add address to bluetooth node Bluetooth doesn't have the mac adress specified
 in eeprom, adding this is needed to get the bluetooth working

---
 arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
index 077a1d37171e..0c23073e6f5c 100644
--- a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
+++ b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
@@ -1038,6 +1038,7 @@ bluetooth: bluetooth {
 		vddrf-supply = <&vreg_l2c_1p3>;
 		vddch0-supply = <&vreg_l11c_3p3>;
 		max-speed = <3200000>;
+		local-bd-address = [ 21 00 00 00 5a ad ];
 	};
 };
 
-- 
2.48.1

